Source/HelixToolkit.Wpf.SharpDX/Shaders/Material.fx

```diff
@@ -25,8 +25,10 @@
 	bool   bHasInstances		= false;
 	bool   bHasShadowMap		= false;
 
-	float2 vShadowMapSize		= float2(1024, 1024);
-	float4 vShadowMapInfo		= float4(0.005, 1.0, 0.5, 0.0);
+	float2 vShadowMapSize = float2(1024, 1024);
+	float4 vShadowMapInfo = float4(0.005, 1.0, 0.5, 0.0);
+
+	float4 vSelectionColor = float4(0.0, 1.0, 1.0, 1.0);

```

Comment:
`vSelectionColor` is added to `MaterialFx` to be used for coloring objects by selection.